cmd/config.go

```diff
@@ -119,8 +120,61 @@ example: cozy-stack config passwd ~/.cozy/
 	},
 }
 
+var genRSAKeysCmd = &cobra.Command{
+	Use:   "gen-rsa-keys [filepath]",
+	Short: "Generate an RSA key pair",
+	Long: `
+cozy-stack config gen-rsa-keys generate a RSA key-pair and save them in the
+specified path. The public key filename is given a ".pub" extension suffix.
+The files permissions are 0400 for private key and 0444 for the public key.
```

Comment:
Can we use 0440 for the public key? I don't see a reason to allow the whole world to read this key.